modules/tile-converter/src/converter-cli.ts

```diff
@@ -36,6 +36,8 @@ type TileConversionOptions = {
    * default: "./deps/egm2008-5.pgm". A model file can be loaded from GeographicLib
    * https://geographiclib.sourceforge.io/html/geoid.html */
   egm: string;
+  /** 3DTiles->I3S only. Whether the converter uses Earth Gravity Model (*.pgm) */
+  noEgm: boolean;
```

Comment:
Maybe just allow egm to be omitted and that means no egm is applied?
```suggestion
   * https://geographiclib.sourceforge.io/html/geoid.html.
   * If not supplied, no EGM correction is applied. */
  egm?: string;
```

Reply:
We're applying default egm in this case, so we needed something different for no-egm

Reply:
egm: 'none' ?

Reply:
Looks much better but I already pushed the PR, so I'll change it later